rust/types/src/markets.rs

```diff
@@ -3,10 +3,14 @@ use serde::{Deserialize, Serialize};
 
 use crate::Blockchain;
 
+/// An asset is most of the time a crypto coin that can have multiple representations
+/// across different blockchains. For example, USDT.
 #[derive(Debug, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct Asset {
+    /// Identifier
     symbol: String,
+    /// See [`Token`]
```

Comment:
[``] is a keyword that allows users to jump into the associated element when clicked